eZ/Publish/Core/MVC/Symfony/Routing/Tests/UrlAliasGeneratorTest.php

```diff
@@ -18,7 +18,7 @@
 use eZ\Publish\Core\MVC\Symfony\SiteAccess;
```

Comment:
Could you also remove `File containing the UrlAliasGeneratorTest class.` like in other files?

Reply:
Also, in others which you touched

Reply:
> Could you also remove `File containing the UrlAliasGeneratorTest class.` like in other files?
> Also, in others which you touched

Aligned via 7856160956.